examples/generate.jl

```diff
@@ -0,0 +1,39 @@
+# helper
+function runcommand(cmd)
+    @info cmd
+    run(cmd)
+end
+
+function generate(dir; execute=true, pluto=true)
+    quote
+        using Pkg
+        Pkg.activate(temp=true)
+        Pkg.add(name="Literate", rev="fe/pluto")
+        using Literate
+
+        const OUTDIR = $dir
+        const INFILE = joinpath(OUTDIR, "notebook.jl")
+
+        # generate pluto notebook:
+        if $pluto
+            TEMPDIR = tempdir()
+            Literate.notebook(INFILE, TEMPDIR, flavor=Literate.PlutoFlavor())
+            runcommand(`mv $TEMPDIR/notebook.jl $OUTDIR/notebook.pluto.jl`)
+        end
+
+        Literate.notebook(INFILE, OUTDIR, execute=false)
+        runcommand(
+            `mv $OUTDIR/notebook.ipynb $OUTDIR/notebook.unexecuted.ipynb`)
+        $execute && Literate.notebook(INFILE, OUTDIR, execute=true)
+
+    end |> eval
+end
```

Comment:
Does this manage state well? We've had some problems at TuringTutorials were some tutorial was failing because of something that happened 4 tutorials back. Also, Julia's problem of environments not being truly separate https://github.com/JuliaLang/julia/issues/35663.

Reply:
Yes, this is just my hack to generate the tutorials. We do have an [intermittent issue](https://discourse.julialang.org/t/execution-of-notebook-in-literate-jl-not-working-but-notebook-executes-fine-in-jupyter-serialization-issue/76387/6) when we try to deserialise models, but it's not consistent. 

Happy to take suggestions on better "low-tech" solution, but perhaps the thing to do is make this a DataScienceTutorial where, as far as I know, we've not had any issues. 

Oh, problem I'm having is that the plots are not appearing when I view the .ipynb in github, even though I see them when I open locally using Jupyter (without re-evaluating).